cli/cmd/clean.go

```diff
@@ -120,15 +118,22 @@ func internalCleanModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
 		status := running.Status(&run)
 		if status.Code == process_utils.ProcessStoppedCode {
 			var statusMsg string
+			var postfix string
+
+			if cmdCtx.Cli.Verbose {
+				postfix = "\n"
+			}
 
-			err := clean(&run)
+			filesAmount, err := clean(&run)
 			if err != nil {
-				statusMsg = "[ERR] " + err.Error()
+				statusMsg = "[ERR] " + err.Error() + postfix
```

Comment:
Yes it would work as you said 
![image](https://github.com/user-attachments/assets/8f628338-aefc-41d8-b603-4b6947ce8259)

Reply:
> Yes it would work as you said

It looks odd.

Reply:
> > Yes it would work as you said
> 
> It looks odd.
Changed it 
![image](https://github.com/user-attachments/assets/4cfda263-c97e-418b-8936-4633bef9c1e6)